### DISCUSSION TITLE
Feb 26

### DISCUSSION THREAD

**Discussion by hanseulhee** [GymHubCommunity/GymHub-FE#219] [DailyMeeting]
### ⚽️ 회의 내용

- [x] 개발 회의

루틴 cud (o)
달에 해당하는 운동기록 조회
스냅샷 crd (0)
회원 정보 수정

각자 하는 거
슬희 - 운동 기록 스냅샷 (get 완), 달에 해당하는 운동기록 조회
윤혁 - 루틴 post, 스냅샷 c
수영 - 루틴 수정 삭제

발표 - 수님

---

스냅샷 = 즐겨찾기 (내가 자주하는 운동 루틴)

**달에 해당하는 운동기록 조회**
   각 날짜마다 무슨 운동을 했는 지 조회
  
**운동기록 스냅샷 무한스크롤 조회**
   저장된 운동에서의 조회

bodyPart에 filter

### 🏀 오늘 할 일

1) 운동기록
2) 해시태그 검색 
3) 마이페이지
4) 달에 해당하는 운동기록 조회
5) 스냅샷
6) 회원정보 수정

스리 - 마이페이지
수영 - 해시태그 검색
윤혁 - 운동기록 post

27일에 플로우 다같이 점검

### 🎱 TMI

스리 - 
궁수 - 
수영 - 
윤혁 -

**Comment by oauch** [comment]
다들 화이팅 🥹

**Reply by hanseulhee** [reply]
화이팅 ~~~~ 🔥🔥🔥🔥
